LINK/usr/bin/periodic_vacuum_full_tables

```diff
@@ -1,97 +1,36 @@
-# so that there is minimal impact to the operating environment (ie, filesystem) 
-# as it is expected that these operations will have a longer duration than
-# the hourly reindex transactions.
-#
-# any single vacuum can be removed by preceding the word "vacuum" with two dashes
-# eg "--vacuum full hosts" will prevent the vacuum from operating on the 'hosts'
-# table
-# 
-# 
-# path_to_psql="/usr/local/bin/psql"                         # Use this path for CFME 2.0
-#path_to_psql="/opt/rh/postgresql92/root/usr/bin/psql"         # Use this path for CFME 3.x 
-path_to_psql="psql"                                           # reduced just to the command when using scl enable
-#
-COMMON_OUTPUT='/var/www/miq/vmdb/log/periodic_continuous_pg_maint_stdout.log'
-#
-echo "current time is $(date) -> periodic vacuum full for selected CFME tables " >> $COMMON_OUTPUT
-#/usr/local/bin/psql  vmdb_production -c "REINDEX TABLE miq_servers "  >> $COMMON_OUTPUT 2>&1
-scl enable postgresql92 -- $path_to_psql  vmdb_production -a  << $$end >> $COMMON_OUTPUT 2>&1
-\timing
-VACUUM full verbose vms ;
-vacuum full verbose binary_blob_parts ;
-vacuum full verbose binary_blobs ;
-vacuum full verbose customization_specs ;
-vacuum full verbose firewall_rules   ;
-Vacuum full verbose hosts ;
-vacuum full verbose storages ;
-vacuum full verbose miq_schedules ;
-vacuum full verbose event_logs ;
+#!/bin/bash
+
+source /etc/default/evm
+LOGFILE=/var/www/miq/vmdb/log/hourly_continuous_pg_maint_stdout.log
 
-vacuum full verbose policy_events ;
-vacuum full verbose snapshots ;
-vacuum full verbose jobs ;
-vacuum full verbose networks ;
-vacuum full verbose miq_queue ;
-vacuum full verbose miq_request_tasks ;
-vacuum full verbose miq_workers;
-vacuum full verbose miq_servers;
-vacuum full verbose miq_searches;
-vacuum full verbose miq_scsi_luns ;
-vacuum full verbose miq_scsi_targets;
-vacuum full verbose storage_files ;
-vacuum full verbose taggings ;
-vacuum full verbose vim_performance_states;
---vacuum full verbose ems_events ;
+echo "current time is $(date) -> periodic vacuum full for selected tables" >> $LOGFILE
 
+psql -U postgres vmdb_production -a <<EOS >> $LOGFILE 2>&1
+\timing
+VACUUM FULL VERBOSE vms;
+VACUUM FULL VERBOSE binary_blob_parts;
+VACUUM FULL VERBOSE binary_blobs;
+VACUUM FULL VERBOSE customization_specs;
+VACUUM FULL VERBOSE firewall_rules;
+VACUUM FULL VERBOSE hosts;
+VACUUM FULL VERBOSE storages;
+VACUUM FULL VERBOSE miq_schedules;
+VACUUM FULL VERBOSE event_logs;
+VACUUM FULL VERBOSE policy_events;
+VACUUM FULL VERBOSE snapshots;
+VACUUM FULL VERBOSE jobs;
+VACUUM FULL VERBOSE networks;
+VACUUM FULL VERBOSE miq_queue;
+VACUUM FULL VERBOSE miq_request_tasks;
+VACUUM FULL VERBOSE miq_workers;
+VACUUM FULL VERBOSE miq_servers;
+VACUUM FULL VERBOSE miq_searches;
+VACUUM FULL VERBOSE miq_scsi_luns;
+VACUUM FULL VERBOSE miq_scsi_targets;
+VACUUM FULL VERBOSE storage_files;
+VACUUM FULL VERBOSE taggings;
+VACUUM FULL VERBOSE vim_performance_states;
+EOS
```

Comment:
Oh cool...and with that change GitHub actually highlights this properly now :smile: